service-front/assets/js/moj/moj.modules/moj.who-are-you.js

```diff
@@ -1,27 +0,0 @@
-// Who Are You module for LPA
-// Dependencies: moj, jQuery
-
-(function () {
-  'use strict';
-
-  moj.Modules.WhoAreYou = {
-
-    init: function () {
-      this.selectionBehaviour();
-    },
-
-    selectionBehaviour: function(){
-      $('#WhoAreYou input:radio:checked').parents('.set').find('.subquestion:first').show();
-
-      $('#WhoAreYou input:radio').change(function (){
-        // Hides all sub-questions...
-        $('#WhoAreYou .subquestion').hide();
```

Comment:
There are no HTML elements with the subquestion class, so this will never fire. Hence more redundant code.